CODIGO/frmOpciones.frm

```diff
@@ -1680,7 +1680,7 @@ scrVolume_Change_Err:
 End Sub
 
 Private Sub VSync_Click()
```

Comment:
Suggestiion, get rid of `VSyncActivado` everywhere in the code. Delete `Private Sub VSync_Click()`